src/Parser/Parser.php

```diff
@@ -3,29 +3,52 @@
 namespace RecursiveTree\Seat\TreeLib\Parser;
 
 use Exception;
+use Illuminate\Support\Collection;
 use RecursiveTree\Seat\TreeLib\Items\EveItem;
 
 abstract class Parser
 {
-    static function parseItems($text, string $EveItemClass = EveItem::class){
+    public static array $parsers = [
+        //fits
+        FitParser::class,
+        // the ingame multibuy
+        MultibuyParser::class,
+        // the old multibuy, but also support prices
+        ManualBuyPrices::class,
+        //also bytes on ingame multibuys, so handle it afterwards
+        NewInventoryWindowParser::class,
+        ItemListParser::class,
+        // Try to parse a mineral scan if nothing else matched
+        MineralScanParser::class
```

Comment:
I don't have a mineral scan at hand, but are you sure your parser bites? The parser goes top down and take the first one that bites. You placed the mineral parser below the item list parser. The item list parser is very aggressive at finding anything, even if it is not a format it knows.

Reply:
I'll test it on my local dev environment and update the PR if needed

Reply:
I just reworked the code and put the mineral scan before the item list parser. It works on my side yes